client/coral-admin/src/components/KarmaTooltip.css

```diff
@@ -0,0 +1,80 @@
+.karmaTooltip {
+  position: relative;
+  display: inline-block;
+  margin: 2px 4px 0;
+}
+
+.icon {
+  font-size: 16px;
+  color: #0D5B8F;
+  -ms-user-select:none;
+  -moz-user-select: none;
+  -webkit-user-select: none;
```

Comment:
👍

Reply:
I'm going to create a separate ticket to address the css vendor prefixes, because we are using them in other places. So we can standardize that as part of the UI refactor.